client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -37,14 +33,19 @@ export const StatsModuleSummaryLinks = props => {
 		}
 	};
 
+	const recordStats = ( item ) => {
+		props.recordGoogleEvent( 'Stats', `Clicked Summary Link: ${ path } ${ item.stat }` );
```

Comment:
Should `path` be `item.path` here?

Alternatively you could destructure `item` too, I suppose